clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AllowLaunchAtomicOperation.groovy

```diff
@@ -94,17 +80,17 @@ class AllowLaunchAtomicOperation implements AtomicOperation<ResolvedAmiResult> {
     // If the AMI was created/owned by a different account, switch to using that for modifying the image
     if (resolvedAmi.ownerId != sourceCredentials.accountId) {
       if (resolvedAmi.getRegion())
-      sourceCredentials = accountCredentialsProvider.all.find { accountCredentials ->
-        accountCredentials instanceof NetflixAmazonCredentials &&
-          ((AmazonCredentials) accountCredentials).accountId == resolvedAmi.ownerId
-      } as NetflixAmazonCredentials
+        sourceCredentials = accountCredentialsProvider.all.find { accountCredentials ->
```

Comment:
unintentional reformat, I will revert

Reply:
actually, this lil guy is missing braces.  I will retain the reformat and add braces

Reply:
:+1: